roaster-selenide/roaster-selenide-api/src/main/java/hu/icellmobilsoft/roaster/selenide/BaseSelenideTestCase.java

```diff
@@ -0,0 +1,61 @@
+/*-
+ * #%L
+ * Coffee
+ * %%
+ * Copyright (C) 2020 i-Cell Mobilsoft Zrt.
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package hu.icellmobilsoft.roaster.selenide;
+
+import org.jboss.weld.environment.se.Weld;
+import org.junit.jupiter.api.BeforeEach;
+import org.openqa.selenium.WebDriver;
+
+import com.codeborne.selenide.WebDriverRunner;
+
+import hu.icellmobilsoft.roaster.weldunit.BaseWeldUnitType;
+
+/**
+ * Base class of Selenide TestCases, what is contained Selenide configuration
+ *
+ * @author speter555
+ * @since 0.2.0
+ */
+public class BaseSelenideTestCase extends BaseWeldUnitType {
+
+    @Override
+    protected void configureWeld(Weld weld) {
```

Comment:
ez már @Deprecated-lett, lehetne az ősbe egy metódus ami extensiont ad hozzá a meglévő weld példányhoz

Reply:
Ezen akartam is kérdezni Imitől, hogy igazából ez meghagyható lenne, hogyha kell még valami a weldhez nyitott maradjon. De max akkor cisnálok egy másik metódust amiben csak Extension-t lehet hozzáadni.

Reply:
Szerintem jobb ha külön metódus van az extension hozzáadásra. Ez a configureWeld(Weld) amugy is olyan bármit is csinalhatok a weld-el volt.

Reply:
Mivel nem kell változás, visszaállítottam az eredetire.